shoop/xtheme/static_src/editor/index.js

```diff
@@ -33,8 +34,11 @@ domready(() => {
         location.href = "?" + newQs;
     });
     $(".layout-add-cell-btn").on("click", function() {
-        const {y} = this.dataset;
```

Comment:
You could just destructure the cell amount (which should be called "cell count" if you ask me) and cell limit into local variables for shorter code:

```javascript
const {y, cellCount, cellLimit} = this.dataset;
if (cellCount >= cellLimit) { // ...
```